pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -223,3 +253,13 @@ func (c *Check) GetObfuscatedStatement(o *obfuscate.Obfuscator, statement string
 func (c *Check) getFullPDBName(pdb string) string {
 	return fmt.Sprintf("%s.%s", c.cdbName, pdb)
 }
+
+func (c *Check) getTagsWithPDB(pdb sql.NullString) []string {
```

Comment:
It's a nit but since the only bit from the check that's needed is the tags, I'd probably just pass the tags in and rename this to `appendXYZ`. 

```suggestion
func appendPDBTag(tags []string, pdb sql.NullString) []string {
```

If we we end up making the pdb tag condition on dbm being enabled, we could keep it with the `(c *Check)` receiver so we can conditionally set it according to whether or not the config is set.